interface-definitions/system_option.xml.in

```diff
@@ -49,6 +49,12 @@
                   <valueless/>
                 </properties>
               </leafNode>
+              <leafNode name="debug-wireguard">
```

Comment:
If we plan to add debug options for more modules, maybe make it `debug wireguard` right away, so that we can add `debug ixgbe` or similar later under the same subtree?

I feel that if we call it `debug-wireguard` now, we set ourselves up for having to write a migration script later.

Reply:
This is actually a good idea to start off with a multi node from the beginning with being wireguard it's first available option.

Reply:
made changes